### ISSUE TITLE
New terms for PMID:15184402

### ISSUE THREAD

**Issue by manulera** [pombase/fypo#4168]
* [x] FYPO:0003329 and FYPO:0005468 should be a child of FYPO:0001585
* [x] New child of FYPO:0003329 (abolished protein localization to cell tip during mitotic interphase)
  * Abolished protein localization to old cell tip during mitotic interphase.
    * A cell phenotype observed in the vegetative growth phase of the life cycle in which the localization of a protein to the **old** cell tip does not occur during interphase of the mitotic cell cycle.
* [x] New child of FYPO:0002031 (abnormal actin cable morphology)
  * Thin? actin cables
    * A physical cellular phenotype observed in the vegetative growth phase of the life cycle in which cells form actin cables that are ???? than normal.
* [x] New child of FYPO:0001019 (monopolar actin cortical patch localization during vegetative growth)
  * monopolar actin cortical patch localization to new end
    * A physical cellular phenotype observed in the vegetative growth phase of the life cycle in which actin cortical patches are localized to only the new end of a cell following cell division.
* [x] New child of FYPO:0000931 (abnormal protein localization to microtubule cytoskeleton during vegetative growth)
  * abnormal protein localization to interphase microtubules
    * A cell phenotype observed in the vegetative growth phase of the life cycle in which the localization of a protein to interphase microtubules is abnormal.
* [X] Remove growing from children of FYPO:0001584? [Made it into a new issue](https://github.com/pombase/fypo/issues/4172)

**Comment by ValWood** [comment]
1.
FYPO:0003329 is already  a child of FYPO:0001585
https://www.pombase.org/term/FYPO:0003329
FYPO:0005468 is increased?

2. I think "thin" is correct as is the opposite of thick, which we have, and must be in PATO because we have a term "thin cell"
You can probably model the logical definitions using the thick actin cables term and the PATO term for "think" from here
thin cell (FYPO:0001879)

3. For "abnormal protein localization to interphase microtubules"
do you think this term should also be "microtubule rather than cytoskeleton?
abolished protein localization to cortical microtubule cytoskeleton during mitotic interphase

4. Agree to remove 'growing' ! 




[abolished protein localization to cortical microtubule cytoskeleton during mitotic interphase](https://www.pombase.org/term/FYPO:0000938)

**Comment by manulera** [comment]
> FYPO:0003329 is already a child of FYPO:0001585

This must be inferred from the reasoner? In protege, it is defined as SubClass of FYPO:0001584, not FYPO:0001585. What should we do in these cases, leave it as it is, or change the asserted subclass?

> FYPO:0005468 is increased?

Yes, my bad

> `thin`
Regarding the "thin", the problem is that what you see in the microscope is that signal is fainter.

>  For "abnormal protein localization to interphase microtubules"
do you think this term should also be "microtubule rather than cytoskeleton?

I will wait on this one for your answers.

It could be more specific and be called "abnormal protein localization to interphase microtubule bundle". I guess for the purpose of the logical definition is better to reference a cellular component rather than saying cytoskeleton?

**Comment by ValWood** [comment]
> the problem is that what you see in the microscope is that signal is fainter, so it's somewhat an interpretation. 

I guess it's OK to say thin. This is reminiscent of the  thicker axial structure annotations in :
https://www.pombase.org/reference/PMID:35333350
which we removed.

I guess there could be other reasons for a stronger signal, but if we go with the author interpretation  of what is happening in the cell it is probably fine (we can remove  if it turns out to be wrong)

**Comment by ValWood** [comment]
I see the authors say "there may be fewer actin filaments present in bud6 actin cables"
and talk about faintness rather than thin-ness but  I think it will be fine.  There does seem to be a massive reduction of actin in the cables  which would presumably make them thinner (and presumably they check that actin levels are not affected, or is that not easy to test in this scenario?)

It might  also be worth adding gloss after the period:
 This is likely due to the presence of fewer actin filaments than normal.
Can also add a related synonym
"reduced number of filaments in actin cables"
(although that doesn't not address your question)
